### PR TITLE
techdocs-common: Release v0.3.4

### PR DESCRIPTION
**Pull request by OrkoHunter**
Release a newer version of `@backstage/techdocs-common` including a bug fix https://github.com/backstage/backstage/pull/4088.

---

Reference documentation (if a package has to be released independently): (Inspiration from @backstage/silver-lining )

1. Remove all changesets inside `.changeset`.
2. Restore changesets that belong to your package.
3. Run `yarn changeset version`.
4. Restore all other changesets, except the ones that belong to your package.
5. Create PR

The version change in the package.json will trigger the publish workflow.

### CODE REVIEW

**Review by emmaindal — APPROVED** [review]

**Comment by Rugvip** [comment]
Meh, so recent release workflow changes broke separate releases. Don't merge this yet and we'll figure out how to fix ^_>

**Comment by OrkoHunter** [comment]
Oh, Okay!

**Comment by OrkoHunter** [comment]
Requesting a review. Please approve when you think it's possible!

**Comment by Rugvip** [comment]
Does having to push a branch that references the merge commit from this PR and then trigger a workflow manually seem like an alright workaround for now? ^_>

**Review by Rugvip — CHANGES_REQUESTED** [review]
To be clear we should really not merge this yet, would put us in a bad state :grin:

**Review by Rugvip — APPROVED** [review]

**Pull request merged**
